pkgs/martinvonz/jj/pkg.yaml

```diff
@@ -4,7 +4,5 @@ packages:
     version: v0.13.0
   - name: martinvonz/jj
     version: v0.4.0
-  - name: martinvonz/jj
-    version: v0.3.1
```

Comment:
Removed the testdata because there is no asset for v0.3.1.
https://github.com/martinvonz/jj/releases/tag/v0.3.1